### DISCUSSION TITLE
GitHub support awfully long reponse time

### DISCUSSION THREAD

**Discussion by jkguttormsen** [community/community#73591] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Has anyone experienced issues lately with the response time of GitHub support? I have active tickets going back to the 29th of August, which haven't been responded to yet. I do each other week or so,  get an email about "We have not forgotten about you", however, it seems like they have. With no other way to reach the support, do you have any suggestions?

**Comment by TheEdaL06** [comment]
I'm sorry to hear about your experience with GitHub support. According to the [GitHub Support documentation](https://docs.github.com/en/enterprise-cloud@latest/support/learning-about-github-support/about-github-support), the standard response time is 24 hours for non-urgent issues, and they offer support in English 24 hours per day, 5 days per week, excluding weekends and national U.S. holidays. 

However, it's important to note that this is a general guideline and actual response times may vary depending on the volume of requests they are handling. 

If you haven't done so already, you might want to consider the following steps:
1. **Check the status of your ticket**: You can view and update your support tickets through the GitHub Support portal.
2. **Escalate your request**: If your issue is urgent or impacting your work, you could try to escalate it by indicating the urgency in your next response to them.
3. **Community Support**: You could also try asking your question on the GitHub Community Forum. The community is quite active and someone might have faced a similar issue and found a solution.

That these are just suggestions and might not guarantee a faster response. I hope your issue gets resolved soon!

**Reply by jkguttormsen** [reply]
I have both checked the status, and tried to "follow up" by responding to my own ticket.

Unfortunately I believe the community aren't able to help. We are trying to upgrade our organization to GitHub Teams, by using teachers benefits from GitHub Global Campus. This doesn't work for our organization, we have "upgraded it", but we are still on GitHub free. I tried to create a new organization, and this worked instantly, so I assume I will need assistant from GitHub support :/

**Reply by KibitoHagoromo** [reply]
Could you please escalate my issue Ticket 4388902 that is still unsolved for one month since the ticket creation.

Best Regards.

**Comment by ozturkyusufziya** [comment]
I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

**Comment by KibitoHagoromo** [comment]
Could you please escalate my issue Ticket 4388902 that is still unsolved for one month since the ticket creation.

Best Regards.

**Comment by jonathanshaheen-verco** [comment]
@github-staff Over 3+ weeks ago I submitted support Ticket ID: 4579923 trying to regain access to my account which was allegedly suspended for terms of service violations, although I was never notified as to what those violations were. Despite multiple replies to that support ticket attempting to get an update on the status of my request, I have yet to hear back from anyone with solutions to my problem. I have been a paying customer of GitHub for over a year and need access to my account for work development. If possible could someone please escalate my Ticket ID to ensure this issue is addressed in a timely manner?
